website/content/docs/installation/source.mdx

```diff
@@ -2,78 +2,81 @@
 layout: docs
 page_title: Installing Vagrant from Source
 description: |-
-  Installing Vagrant from source is an advanced topic and is only recommended
-  when using the official installer is not an option. This page details the
-  steps and prerequisites for installing Vagrant from source.
+  Vagrant installations from source is an advanced operation. It is only recommended
+  if you cannot use the official installer.
 ---
 
-# Installing Vagrant from Source
+# Install Vagrant from source
 
-Installing Vagrant from source is an advanced topic and is only recommended
-when using the official installer is not an option. This page details the
-steps and prerequisites for installing Vagrant from source.
+Vagrant installations from source is an advanced operation. We only recommended
+if you cannot use the official installer. This page details the prerequisites and three
+steps to install Vagrant from source.
+
+1. Install Ruby
+1. Clone Vagrant
+1. Configure locally
 
 ## Install Ruby
 
-You must have a modern Ruby (>= 2.2) in order to develop and build Vagrant. The
-specific Ruby version is documented in the Vagrant's `gemspec`. Please refer to
-the `vagrant.gemspec` in the repository on GitHub, as it will contain the most
-up-to-date requirement. This guide will not discuss how to install and manage Ruby.
-However, beware of the following pitfalls:
+You must install Ruby 2.7 or newer to develop and build Vagrant. The
```

Comment:
Currently Vagrant needs Ruby between 3.0 - 3.2. This will keep changing, so this can probably be a bit more vague and continue to reference the `vagrant.gemspec` to provide the most up to date info on Ruby version.